studio/settings.py

```diff
@@ -387,3 +393,72 @@
 # Specifically, apps.tests.test_user_has_no_access was failing.
 # Also anonymous access to pages was not working.
 ANONYMOUS_USER_NAME = None
+
+
+LOGGING = {
+    "version": 1,
+    "disable_existing_loggers": False,
+    "formatters": {
+        "json_formatter": {
+            "()": structlog.stdlib.ProcessorFormatter,
+            "processor": structlog.processors.JSONRenderer(),
+        },
+        "plain_console": {
+            "()": structlog.stdlib.ProcessorFormatter,
+            "processor": structlog.dev.ConsoleRenderer(),
+        },
+        "key_value": {
+            "()": structlog.stdlib.ProcessorFormatter,
+            "processor": structlog.processors.KeyValueRenderer(key_order=["timestamp", "level", "event", "logger"]),
+        },
+        "colored": {
+            "()": colorlog.ColoredFormatter,  # colored output
+            "datefmt": "%Y-%m-%d %H:%M:%S",
+            "format": "%(log_color)s%(asctime)s - %(levelname)s - %(module)s: %(message)s%(reset)s",
+        },
+    },
+    "handlers": {
+        "console": {
+            "class": "logging.StreamHandler",
+            "formatter": "colored",
+        },
+        "json_file": {
+            "class": "logging.handlers.WatchedFileHandler",
+            "filename": "logs/json.log",
+            "formatter": "json_formatter",
+        },
+        "flat_line_file": {
+            "class": "logging.handlers.WatchedFileHandler",
+            "filename": "logs/flat_line.log",
+            "formatter": "key_value",
+        },
+    },
+    "loggers": {
+        "django_structlog": {
+            "handlers": ["json_file"],
+            "level": "DEBUG" if DEBUG else "WARNING",
+        },
+        # Make sure to replace the following logger's name for yours
+        "portal": {
+            "handlers": ["console"],
+            "level": "DEBUG" if DEBUG else "WARNING",
+        },
+    },
+}
```

Comment:
I would personally (and that's important, as we'd better discuss it I think) go for "INFO" level for production and "DEBUG" for development.

The Idea would be that INFO level would be used by default. But if you would be debugging on your machine, you could use debug level.

That's how I feel about it at least